app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -0,0 +1,183 @@
+module Katello
+  module Pulp3
+    class RepositoryMirror
```

Comment:
Is some one extending this or using this?

Reply:
n/m I guess the diff didnt load for me

Reply:
we answered this in person